packages/koenig-lexical/src/hooks/usePreviousFocus.js

```diff
@@ -0,0 +1,24 @@
+import {useState} from 'react';
+
+export const usePreviousFocus = (onClick, name) => {
+    const [previousRange, setPreviousRange] = useState(null);
```

Comment:
I would make this a ref instead of state to avoid extra re-renders

Reply:
great point, updated in https://github.com/TryGhost/Koenig/pull/680/commits/495aa25105f34bbb5f952aeb37ec31f61ac08da3